app/extensions/edm/__init__.py

```diff
@@ -88,6 +91,21 @@ class EDMManager(RestManager):
     def __init__(self, pre_initialize=False, *args, **kwargs):
         super(EDMManager, self).__init__(pre_initialize, *args, **kwargs)
 
+    def version_check(self):
+        edm_version = current_app.edm.get_dict('version.dict', None)
+        if edm_version is None or 'date' not in edm_version:
+            log.error('could not determine EDM version')
+            return False
+        if edm_version['date'] >= self.VERSION_REQUIRED:
+            log.info(
+                f"EDM version check passed: edm_version={edm_version['date']}  >=  version_required={self.VERSION_REQUIRED}"
+            )
+            return True
+        log.error(
+            f"EDM version check FAILED: edm_version={edm_version['date']}  <  version_required={self.VERSION_REQUIRED}"
+        )
+        return False
+
```

Comment:
What is the bug with the version check?

Reply:
bug is now resolved via PR155 - so this is technically all working now.  i have adjusted the tests accordingly.